data/testimonials.yaml

```
@@ -0,0 +1,130 @@
+    industry: "FinTech"
+    project_type: "MVP Development"
+    engagement_duration: "12 months"
+    team_size: "4 developers"
+    description: "JetThoughts transformed our initial concept into a fully functional fintech platform. Their expertise in both technical implementation and product strategy was invaluable. The team delivered on time and within budget, while maintaining the highest quality standards."
+    results:
+      - "Launched MVP 3 months ahead of schedule"
+      - "Secured $2M Series A funding"
+      - "Achieved 10,000+ active users within 6 months"
+    technologies: ["Ruby on Rails", "Vue.js", "PostgreSQL", "Stripe API"]
+    location: "San Francisco, USA"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)